packages/core-interop/src/test/Localization.test.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import sinon from "sinon";
+import { Localization } from "@itwin/core-common";
+import { LOCALIZATION_NAMESPACE } from "@itwin/presentation-hierarchy-builder";
+import { createLocalizationFunction } from "../core-interop/Localization";
+
+describe("createTranslator", () => {
+  it("creates a localization function using provided core `Localization` object", async () => {
+    const localization = {
+      getLocalizedString: (input: string) => `${input}_localized`,
+      registerNamespace: async () => {},
+    } as unknown as Localization;
+    const spy = sinon.spy(localization, "registerNamespace");
```

Comment:
could just set the spy right away:
```ts
    const localization = {
      getLocalizedString: (input: string) => `${input}_localized`,
      registerNamespace: sinon.spy(),
    } as unknown as Localization;
```

Reply:
Changed to:
```ts
const registerNamespaceSpy = sinon.spy();
const localization = {
  getLocalizedString: (input: string) => `${input}_localized`,
  registerNamespace: registerNamespaceSpy,
} as unknown as Localization;
const translator = await createLocalizationFunction(localization);
expect(registerNamespaceSpy).to.be.calledWith(LOCALIZATION_NAMESPACE);
```
Because I am checking what the spy was called with.